ev-dev-tools/src/ev_cli/__init__.py

```diff
@@ -1,2 +1,2 @@
 """EVerest command line utility."""
-__version__ = '0.2.1'
+__version__ = '0.2.2'
```

Comment:
I'm just wondering whether this version shouldn't be in sync with the repo tags?